pkg/types/types.go

```diff
@@ -275,6 +275,8 @@ type Options struct {
 	ShowMatchLine bool
 	// EnablePprof enables exposing pprof runtime information with a webserver.
 	EnablePprof bool
+	// Include all http request, response chain in json|l output
```

Comment:
```suggestion
	// Include all requests, response chains in JSON line output
```

Since this isn't just limited to HTTP-protocol-based.